lib/datadog/tracing/span_operation.rb

```diff
@@ -35,7 +35,7 @@ class SpanOperation
         :start_time,
         :trace_id,
         :type
-      attr_accessor :links, :status
+      attr_accessor :links, :status, :span_events
```

Comment:
It is a bit unfortunate that we have both `events` and `span_events` for this object.

Reply:
Yup 😞

Reply:
Should some documentation be added explaining what `events` and `span_events` are to prevent confusion?

Reply:
Should some documentation be added explaining what `events` and `span_events` are to prevent confusion?